vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -173,6 +173,11 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 			Computed:    true,
 			Description: "A flag internal to Terraform that indicates that this resource was either imported or came from a earlier major version of this resource.",
 		},
+		"moid": &schema.Schema{
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "",
```

Comment:
Can you add a description to this?

Reply:
Yeah sorry! My mistake! Already update it some minutes ago. :-)